src/main/java/frc/robot/subsystems/gripper/Gripper.java

```diff
@@ -51,8 +54,14 @@ public boolean atSetpoint() {
                 inputs.angleSetpoint, GripperConstants.THRESHOLD.in(Units.Percent));
     }
 
+    public boolean isReversed() {
```

Comment:
The function name is too general, it's not clear what "reversed" means.

Reply:
The gripper is reversed meaning it is backwards

Reply:
In a second though that's a good name. However, I think the implementation is too restrictive -- why not just check if the gripper is pointing somewhere backward, instead of checking if it's near one specific angle?

Reply:
a13f863